Scripts/utils/calc_noise.py

```diff
@@ -0,0 +1,78 @@
+from math import log10
+
+import parameters.assignment as param
+
+class NoiseModel:
+    """Model for calculating noise zone width for road links.
+
+    Parameters
+    ----------
+    morning_network : inro.emme.network.Network
+        Network used for morning peak hour assignment
+    light_modes : tuple of str
+        Extra attribute names for 24-h traffic per assignment class
+    heavy_modes : tuple of str
+        Extra attribute names for 24-h traffic per assignment class
+    """
```

Comment:
If you have a source for these noise calculations formulas, this would be a great place to cite it. :)